EEP_Inventar.html

```diff
@@ -0,0 +1,255 @@
+			const PosY		= Gleis.getElementsByTagName('Dreibein')[0].getElementsByTagName('Vektor')[0].getAttribute('y') / 100;
+
+			// Weichen
+			if (WeicheID) {
+				htmlAppend('p', 
+					`Weiche ${WeicheID}, 
+					 ${GleissystemText[GleissystemID]},
+					 ${GleisID},
+					 x = ${PosX.toFixed(0)} m,
+					 y = ${PosY.toFixed(0)} m
+					`
+				);
+			}
+			
+		} // Gleise
+	} // Gleissysteme
+
+
+
+	// Signale
+	htmlAppend('h4', `Signale`); 
+	for (const Gleissystem of Gleissysteme) {
+		const GleissystemID 	= Gleissystem.getAttribute('GleissystemID');
+//		htmlAppend('p', `${GleissystemText[GleissystemID]}`);
+		
+		const Gleise = Gleissystem.getElementsByTagName('Gleis');
+		for (const Gleis of Gleise) {
+
+			const GleisID 	= Gleis.getAttribute('GleisID');
+			//const clsid 	= Gleis.getAttribute('clsid');		// Gleisart
+			//const stil 		= Gleis.getAttribute('stil');		// Gleisstil
+			//const gsbname 	= Gleis.getAttribute('gsbname');	// Dateiname
+			//const WeicheID	= Gleis.getAttribute('Key_Id'); 	// Id einer Weiche
+			
+			//const Laenge	= Gleis.getElementsByTagName('Charakteristik')[0].getAttribute('Laenge') / 100;
+			//const Kruemmung	= Gleis.getElementsByTagName('Charakteristik')[0].getAttribute('Kruemmung') * 100;
+			const PosX		= Gleis.getElementsByTagName('Dreibein')[0].getElementsByTagName('Vektor')[0].getAttribute('x') / 100;
+			const PosY		= Gleis.getElementsByTagName('Dreibein')[0].getElementsByTagName('Vektor')[0].getAttribute('y') / 100;
+
+			const Meldungen = Gleis.getElementsByTagName('Meldung');
+			for (const Meldung of Meldungen) {
+
+				const SignalID	= Meldung.getAttribute('Key_Id'); 	// Id eines Signals
+				const Name		= Meldung.getAttribute('name');
+
+
+				// Signale
+				if (SignalID) {
+					htmlAppend('p', 
+						`Signal ${SignalID}, 
+						 ${GleissystemText[GleissystemID]},
+						 ${GleisID},
+						 x = ${PosX.toFixed(0)} m,
+						 y = ${PosY.toFixed(0)} m,
+						 ${Name}
+						`
+					);
+				}
+			
+			}
+			
+		} // Gleise
+	} // Gleissysteme
+
+
+}
+	
+function htmlAppend(tagName, innerHTML) {
+	var element = document.createElement(tagName);
+	element.innerHTML = innerHTML;
+	document.getElementById('container').appendChild(element);
+}
+
+</script>
+
+<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+<script> // html page
+
+</script>
+
+<style> /* html page */
+.hidden {
+	display: none;
+}
+
+#container p {
+	margin: 0;
+}
+
+</style>
+
+</head>
+
+<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+<body>
+
+<h3 id='filename'>EEP Inventar</h3>
+
+<div id='header'> <!-- The header contains the initial selection screen -->
+
+<div id='fileselector'>
+<form action='#' onsubmit="return false;"> <!-- better not to use inline listener, explizit register is better --> 
+	<label for="fileinput">Wähle und lade eine EEP-Anlage-Datei (.anl3)</label>
+	<input type='file' id='fileinput' accept='.anl3' onchange="loadFile(this.files)">
```

Comment:
You are passing `this.files` into `loadFile` but it is never used.
You could refactor `loadFile` to use this parameter like I did it in the other file.

Reply:
Erledigt (das hat man davon, wenn man eine kurze, jedoch alter Version des anderen Programms als Vorlage nutzt)